examples/todomvc/src/main/scala/example/TodoList.scala

```diff
@@ -77,7 +77,7 @@ object TodoList {
                 onCancelEditing = editingDone(),
                 todo = todo,
                 isEditing = editing.contains(todo.id)
-              )))
+              ))).toTagMod
```

Comment:
Hi @alonsodomin, you made wrong merge here :( Please remove `.toTagMod` on this line, because there is right one (on line 70)